packages/react/src/evm/components/MediaRenderer.tsx

```diff
@@ -395,6 +405,44 @@ const LinkPlayer = React.forwardRef<HTMLAnchorElement, MediaRendererProps>(
 
 LinkPlayer.displayName = "LinkPlayer";
 
+const ModelViewer = React.forwardRef<HTMLCanvasElement, MediaRendererProps>(
+  ({ src, alt, style, ...restProps }, ref) => {
+    const [loaded, setLoaded] = useState(false);
+    const modelRef = useRef<HTMLCanvasElement>(null);
+  
+    useEffect(() => {
+      loadModelViewer();
+    },[])
+  
+    const loadModelViewer = () => {
+      const existingScript = document.getElementById('modelViewer');
+      if (!existingScript) {
+        const script = document.createElement('script');
+        script.src = 'https://unpkg.com/@google/model-viewer/dist/model-viewer.min.js';
```

Comment:
would prefer to dynamically load this from `node_modules` (bundle it in but as a code-split file using dynamic `import()`)